kr8s/_objects.py

```diff
@@ -828,6 +830,30 @@ async def uncordon(self) -> None:
         """
         await self.async_patch({"spec": {"unschedulable": False}})
 
+    async def taint(self, key: str, value: str, *, effect: str) -> None:
+        """Taint a node."""
```

Comment:
Given that we are operating based on remote state we probably want to refresh the state before we continue.

```suggestion
        """Taint a node."""
        await self.async_refresh()
```